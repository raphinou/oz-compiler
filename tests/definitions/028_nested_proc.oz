% global A is redeclared by T, and the argument passed to the call of T is that global.
local
   A P B
in
   proc {P V}
      T W
   in
      proc {T U}
         A=44
      in
         {Show A}
         {Show U}
      end
      proc {W}
         {Show A}
      end
      {W}
      {T A}
   end
   A = 5
   B = 7
   {P A}
end
