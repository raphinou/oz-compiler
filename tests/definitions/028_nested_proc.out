5
44
5
